src/test/regress/sql/multi_utilities.sql

```diff
@@ -198,3 +195,85 @@ SELECT worker_create_or_alter_role(NULL, 'create role dontcrash', NULL);
 
 -- confirm that citus_create_restore_point works
 SELECT 1 FROM citus_create_restore_point('regression-test');
+
+SET citus.shard_count TO 1;
+SET citus.shard_replication_factor TO 1;
+SET citus.next_shard_id TO 970000;
+
+SET citus.log_remote_commands TO OFF;
+
+CREATE TABLE local_vacuum_table(id int);
+
+CREATE TABLE reference_vacuum_table(id int);
+SELECT create_reference_table('reference_vacuum_table');
+
+CREATE TABLE distributed_vacuum_table(id int);
+SELECT create_distributed_table('distributed_vacuum_table', 'id');
+
+SET citus.log_remote_commands TO ON;
+
+-- should propagate to all workers because no table is specified
+VACUUM;
```

Comment:
vacuum has quite a few options
https://www.postgresql.org/docs/current/sql-vacuum.html

are these all supported? should we add some tests?

Reply:
* I have extended regression tests for vacuum. Only 1 option(PROCESS_TOAST) could not be tested for pg13 because no such option in pg13, I currently commented that test out. **What do you suggest to test that option only for pg14?**

* I have added SKIP_LOCKED support. Previously deadlock happened if a restricted lock is obtained for a relation in another transaction while 'vacuum full' is executing. Currently, we just skip that relation, when the lock is not available and skip_locked is enabled. I have added isolation test for that.

* There was a deadlock bug when we run unqualified vacuum on a 1 node citus cluster. Now fixed.